src/com/shop/Shop.java

```diff
@@ -1,324 +1,335 @@
+					System.out.println("Product id " + productId + " quantity 1 was returned.");
+				}
+			} else {
+				System.out.println("Product id " + productId + " was NOT ordered.");
+			}
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+	}
+
+	private void menuS() {
+		System.out.println("Enter Product Full or Partial Name: ");
+		String productName = input.next();
+		System.out.println();
+
+		try {
+			selectProductsName.setString(1, "%" + productName + "%");
+			queryResult = selectProductsName.executeQuery();
+			System.out.printf("%-5s | %-20s | %-10s%n", "ID", "NAME", "PRICE");
+			
+			while (queryResult.next()) {
+				System.out.printf("%-5d | %-20s | %.2f%n", queryResult.getInt("pid"), queryResult.getString("name"),
+						queryResult.getDouble("price"));
+			}
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+	}
+
+	private void menuE() {
+		try {
+			selectTotalCost.setInt(1, id);
+			queryResult = selectTotalCost.executeQuery();
+			System.out.printf("%-5s | %-20s | %-10s%n", "ID", "NAME", "TOTAL");
+			
+			while (queryResult.next()) {
+				System.out.printf("%-5d | %-20s | %-10.2f%n", queryResult.getInt("pid"), queryResult.getString("name"),
+						queryResult.getDouble("total"));
+			}
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+	}
+
+	private void menuB() {
+		try {
+			selectCustomerRow.setInt(1, id);
+			queryResult = selectCustomerRow.executeQuery();
+			queryResult.next();
+			System.out.printf("Budget is: %.2f%n", queryResult.getDouble("budget"));
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+	}
+
+	private void createCustomer() {
+		System.out.println("Enter Name: ");
+		String customerName = input.next();
+		System.out.print("Enter Budget: ");
+		double customerBudget = input.nextDouble();
+		System.out.println();
+
+		try {
+			queryResult = selectAllCustomers.executeQuery();
+			do {
+				if (!queryResult.next()) {
+					break;
+				}
+				id = queryResult.getInt("cid") + 1;
+			} while (true);
+
+			insertCustomerRow.setInt(1, id);
+			insertCustomerRow.setString(2, customerName);
+			insertCustomerRow.setDouble(3, customerBudget);
+			queryResult = insertCustomerRow.executeQuery();
+
+			System.out.println("===== New Customer Was Created. =====");
+			System.out.println("Customer ID: " + id + " | Name: " + customerName + " | Budget: " + customerBudget + "\n");
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+	}
+
+	public static void main(String[] args) {
+		Shop shop = new Shop();
+		String menuOption = "";
+		Connection conn = shop.connect();
+		
+		if (conn != null) {
+			shop.setPreparedStatements(conn);
+			do {
+				System.out.print("Enter Customer ID: ");
+				shop.id = shop.input.nextInt();
+				
+				if (shop.id == -1) {
+					shop.createCustomer();
+					break;
+				}
+			} while (!shop.isCustomerVerified());
+
+			while (!menuOption.equalsIgnoreCase("x")) {
+				shop.displayMenu();
+				menuOption = Character.toString(shop.input.next().charAt(0));
```

Comment:
Using String instead of char type to allow for usage of String equalsIgnoreCase()